integration_tests/backend/test_backend.py

```diff
@@ -0,0 +1,81 @@
+from collections import defaultdict
+
+import pytest
+import requests
+from setup.load_workflow import create_test_endpoint_GetWorkflowTables_flow
+
+import aqueduct
+
+
+class TestBackend:
+    @classmethod
+    def setup_class(cls):
+        cls.client = aqueduct.Client(pytest.api_key, pytest.server_address)
+        cls.flows = []
+
+        # For test_endpoint_GetWorkflowTables
+        cls.test_endpoint_GetWorkflowTables_flow_table_names = [
+            "table_1",
+            "table_1",
+            "table_1",
+            "table_2",
+        ]
+        cls.test_endpoint_GetWorkflowTables_flow_update_modes = [
+            "append",
+            "append",
+            "replace",
+            "append",
+        ]
+        cls.test_endpoint_GetWorkflowTables_flow = create_test_endpoint_GetWorkflowTables_flow(
+            cls.client,
+            "test_endpoint_GetWorkflowTables flow",
+            cls.test_endpoint_GetWorkflowTables_flow_table_names,
+            cls.test_endpoint_GetWorkflowTables_flow_update_modes,
+        )
+        cls.flows.append(cls.test_endpoint_GetWorkflowTables_flow)
+
+    @classmethod
+    def teardown_class(cls):
+        for flow in cls.flows:
+            cls.client.delete_flow(flow.id())
+
+    def test_endpoint_GetWorkflowTables(self):
+        headers = {"api-key": pytest.api_key}
+        url = f"{pytest.adapter}{pytest.server_address}/api/workflow/{self.test_endpoint_GetWorkflowTables_flow.id()}/tables"
```

Comment:
I feel like its more readable to have a URL template, like what we do in api_client.py. That way other tests that use the same route can reuse the same template without drift.